org-wild-notifier.el

```diff
@@ -361,17 +361,18 @@ Do nothing if a check is already in progress in the background."
   (unless (and org-wild-notifier--process
                (process-live-p org-wild-notifier--process))
     (setq org-wild-notifier--process
-          (async-start
-           (org-wild-notifier--retrieve-events)
-           (lambda (events)
-             (setq org-wild-notifier--process nil)
-             (-each
-                 (->> events
-                      (-map 'org-wild-notifier--check-event)
-                      (-flatten)
-                      (-uniq))
-               'org-wild-notifier--notify)
-             (setq org-wild-notifier--last-check-time (current-time)))))))
+          (let ((default-directory "~/"))
```

Comment:
Can we use `user-emacs-directory` instead?

Reply:
Sure, done!